src/common/types.ts

```diff
@@ -18,6 +18,7 @@ export interface ILinterConfig {
     column: number,
     message: Array<number|string> | number,
     security: number
+    file: string
```

Comment:
`file` seems not used?

Reply:
Yeah that's right, I was testing some things.. I will remove it.